Changelog.md

```diff
@@ -32,6 +32,7 @@
 - Added background job to clean tmp folder (#5963)
 - Removed consideration of due date from scanned exam (#5964)
 - Added Exception Notification Gem (#5965)
+- Resolved issue 5905 unintuitive submissions UI design (#6001)
```

Comment:
For new features (not bug fixes), Changelog descriptions can mention issues, but should really focus on what the change actually was, not what problem was solved. (i.e., say the "what", not the "why")